neo4j/internal/testutil/poolfake.go

```diff
@@ -32,17 +33,17 @@ type PoolFake struct {
 	CleanUpHook func()
 }
 
-func (p *PoolFake) Borrow(ctx context.Context, serverNames []string, wait bool, _ log.BoltLogger) (db.Connection, error) {
+func (p *PoolFake) Borrow(context.Context, []string, bool, log.BoltLogger, time.Duration) (db.Connection, error) {
 	return p.BorrowConn, p.BorrowErr
 }
 
-func (p *PoolFake) Return(ctx context.Context, c db.Connection) {
+func (p *PoolFake) Return(context.Context, db.Connection) {
 	if p.ReturnHook != nil {
 		p.ReturnHook()
 	}
 }
 
-func (p *PoolFake) CleanUp(ctx context.Context) {
+func (p *PoolFake) CleanUp(context.Context) {
```

Comment:
In this PR, I've unnamed found unused arguments (like here) and `_`. which one is the preferred style?

Reply:
`_` is used when there are named parameters
if none of the parameters are named, names can be scratched completely (have you seen an instance where all of the names were `_`?)

Reply:
I just checked, this PR only uses `_` to ignored part of the returned results or part of the parameters, which is the way go...
when all parameters are ignored like above, the name part is completely omitted